routine-data.h

```diff
@@ -2,29 +2,44 @@
 #define __ROUTINE_DATA__
 
 using getData = int();
+using Filter = int();
 
 void gatherData(getData func) {  
   int d = func();
   publishInt("data", d);
-  data[current_sample++] = d;
+  data[current_sample] = d;
+  Serial.println("Gather data:");
+  for (int i = 0; i < DATA_SIZE; ++i)
+  {
+    Serial.print(data[i]);
+    Serial.print("; ");  
+  }
+  Serial.print("Current_sample = ");
+  Serial.println(current_sample++);
```

Comment:
```suggestion
current_sample++;
  Serial.println(current_sample);
```

1. change current sample
2. move that after line 10
3. move serial logging to separate function and call it out of gatherData function

avoid modifying in Serial.print - because evrth will not work if I just comment logging - and I'll never figure out the reason!!!!